plugin/trino-memsql/src/test/java/io/trino/plugin/memsql/TestMemSqlDistributedQueries.java

```diff
@@ -120,6 +120,11 @@ protected TestTable createTableWithDefaultColumns()
             return Optional.empty();
         }
 
+        if (typeName.contains("char(1)")) {
+            // https://github.com/trinodb/trino/issues/6746
```

Comment:
Please add the same link to the varchar branch above.

Reply:
I did not understand the explanation. Do you mean that MemSQL's `varchar(1)` is case sensitive (by default), while `char(1)` is not (by default)?